scripts/scripts.js

```diff
@@ -229,14 +229,8 @@ async function loadEager(doc) {
   let pageType = 'CMS';
   if (document.body.querySelector('main .product-details')) {
     pageType = 'Product';
-    const sku = getSkuFromUrl();
-    window.getProductPromise = getProduct(sku);
```

Comment:
I believe `getProduct` is no longer used. This function should be removed.

Reply:
It should be removed. It's only used on "product-details-custom" which is a different block.